### DISCUSSION TITLE
blog/entity-framework-core-in-aspnet-core/

### DISCUSSION THREAD

**Discussion by giscus** [codewithmukesh/discussions#48] [General]
# blog/entity-framework-core-in-aspnet-core/

In this article, we will go through Entity Framework Core in ASP.NET Core 3.1 along with its various implementations and concepts.

https://codewithmukesh.com/blog/entity-framework-core-in-aspnet-core/

**Comment by rajaraok9** [comment]
Hi Mukesh,

I am new to ASP.Net core, your article is really good. I would like to learn complete Dotnet from you, I will pay the fees. Send me a message on rajaraok9@gmail.com

**Comment by rajaraok9** [comment]
Excellent Mukesh. Keep it up.

**Comment by YashSmithh** [comment]
ASP.NET Core developers can accelerate productivity with top libraries like AutoMapper for object mapping, Serilog for logging, Polly for resilience, and Swashbuckle for API documentation. Others include Hangfire, IdentityServer, FluentValidation, and MediatR. At dev technosys, we harness these libraries for robust software and <a href="https://devtechnosys.com/car-rental-app-development.php">car rental app development</a> ensuring scalable, high-performance applications.
